mp/src/game/shared/in_buttons.h

```diff
@@ -33,10 +33,11 @@
 #define IN_ZOOM			(1 << 19)	// Zoom key for HUD zoom
 #define IN_WEAPON1		(1 << 20)	// weapon defines these bits
 #define IN_WEAPON2		(1 << 21)	// weapon defines these bits
-#define IN_BULLRUSH		(1 << 22)   // used for displaying when bhop is disabled
+#define IN_BHOPDISABLED	(1 << 22)	// used for displaying when bhop is disabled
 #define IN_GRENADE1		(1 << 23)	// grenade 1
 #define IN_GRENADE2		(1 << 24)	// grenade 2
 #define	IN_ATTACK3		(1 << 25)
 #define IN_STRAFE       (1 << 26)
+#define IN_PAINT        (1 << 27)
```

Comment:
A note about this:

https://github.com/momentum-mod/game/blob/fa6f7ab6c8867798d14cd876f81d2c979f1661ac/mp/src/game/shared/momentum/run/mom_replay_data.h#L7-L9

Until we do the refactor, whenever they paint in replays it'll be stored as a teleported frame. We definitely don't want that.

Reply:
Good catch, although I really, really hate that that's a thing 😠
Changed IN_PAINT to take IN_GRENADE2's spot and removed all references to IN_GRENADE2